src/AlgAss/Ramification.jl

```diff
@@ -136,15 +136,14 @@ function schur_index(A::AbsAlgAss{nf_elem}, P::InfPlc)
   @req iscentral(A) "Algebra must be central"
   @req issimple(A) "Algebra must be simple"
 
-  if dim(A) % 4 != 0
-    return 1
-  end
+  dim(A) % 4 == 0 && is_real(P) || return 1
```

Comment:
Hm, this check does not look right. We have `dim(A) % 4 != 0 || !is_real(P)` implies Schur index equal one. The new check would fail for a quaternion algebra?

Reply:
It is just a matter of perspective to say either $¬a ∨ ¬b ⇒ \text{return }1$ or $a ∧ b \ ∨ \ \text{return }1$. I prefer the latter, as it means that afterwards $a∧b$ holds, which, in my eyes, helps understanding the code that follows.

Reply:
By the way, `schur_index(Hecke.quaternion_algebra2(QQ, -1, -1)) == 2` works fine with my code. This should also be covered by the existing tests for the `schur_index` which run successfully.

Reply:
Ah sorry, you are of course right. I parsed this wrong with `&&` at the end (that is, if the first part holds, then do this).